clients/ts/signalr/src/browser-index.ts

```diff
@@ -5,4 +5,20 @@
 
 import "es6-promise/dist/es6-promise.auto.js";
 
+// Copy from Array.prototype into Uint8Array to polyfill on IE. It's OK because the implementations of indexOf and slice use properties
+// that exist on Uint8Array with the same name, and JavaScript is magic.
+// We make them 'writable' because the Buffer polyfill messes with it as well.
+if (!Uint8Array.prototype.indexOf) {
```

Comment:
Suggested by Mozilla as the polyfill: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/slice#Polyfill

And it works, so 🤷‍♂️